### PR TITLE
Update section on xref links

### PR DESCRIPTION
**Pull request by gewarren**
cc @IEvangelist

### CODE REVIEW

**Comment by gewarren** [comment]
#sign-off

**Review by IEvangelist — APPROVED** [review]

File: Contribute/content/how-to-write-links.md [hunk 1]
```diff
@@ -155,10 +155,10 @@ To go to a section on another page.
 
 ## XRef (cross reference) links
 
-XRef links are the recommended way to link to APIs, because they're validated at build time. To link to auto-generated API reference pages in the current docset or other docsets, use XRef links with the unique ID ([UID](#determine-the-uid)) of the type or member.
+XRef links are the recommended way to link to APIs, because they make it easy to customize the link text. Additionally, they're validated at build time, and if the URL to the API reference were to change, the link would still work. To link to auto-generated API reference pages in the current docset or other docsets, use XRef links with the unique ID ([UID](#determine-the-uid)) of the type or member.
 
 > [!TIP]
-> You can use the [Learn Markdown extension for VS Code][docsextension] (part of the Learn Authoring Pack) to insert .NET XRref links that are surfaced in the [.NET API Browser][].
+> The [API Reference Link Helper extension for VS Code](https://github.com/IEvangelist/xref-helper) makes it super easy to insert .NET API XRref links into Markdown and XML files.
```
Comment (IEvangelist) [inline Contribute/content/how-to-write-links.md:161]:
```suggestion
> The [API Reference Link Helper extension for VS Code](https://github.com/IEvangelist/xref-helper) makes it super easy to insert .NET API Xref links into Markdown and XML files.
```

**Comment by gewarren** [comment]
Ping @jehchow

**Comment by jehchow** [comment]
@gewarren I'm no longer managing this repo - @mikekinsman Can you assist?

**Review by mikekinsman — APPROVED** [review]

**Pull request merged**